### ISSUE TITLE
unBalance Tools in 9.0

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#13021]
I’ll start with such tools as a hoe and a scythe.They directly compete with a vehicle such as Steam Tractor.
In stages of evolution, It happens approximately like this:
1. You have a wooden hoe and stone scythe.
2. After receiving the smelting, you can make yourself Iron scythe and Iron Hoe.
3. At this stage, the Mechanic develops with might and main, but in parallel with the mechanic, an advanced smelting can swing in full swing.

Therefore, if you set a goal for a tractor and a steel tool, then they will appear at same time. However, Steam tractor is a specific pleasure. Low count of people use steam tractor now. As a result, we have that the Tractor competes with a steel scythe and a steel hoe.
The tractor even in version 8.3.0 has limited use: Mainly for large fields with "one collecting" plants(as Corn, Wheat).
After the tool changes planned in 9.0.0, Steam tractor becomes just garbage. Why should I use a tractor now? And what is the most incomprehensible, what should be added to Steam tractor so that it's same effectivity as the tool?
For me It would be enough that tools Tier 1 and Tier 2 processes 1 cell, Tier 3 and Tier 4 the tool processes 2x2 cells. 

Same thing with machete. It's **very** good now. For the destruction of plants, I think at the last tier 3x3 cells will be enough. But I would stop 2x2.

Nothing really can be said about the hammer. From what I see: It looks very interesting, but so far it will not work as it should be, It's very difficult to conclude.

I think it’s not necessary to release a remake tool without the balance of other objects with which this tool competes. No need to do super items.
 
As an example. Players didn't like the new redesigned realistic excavator in patch 0.8.2.0. The players said that they didn't need realistic, they needed to dig on it conveniently and quickly. But it was said that it isn't realistic to dig of scoop side. It has become useless. Mirasrael remade it a lot so that Exca and Skid Steer was competitively capable.
And what now? Now we will have a realistic hand tool that mows or processes 25 square meters in 9.0.
If you add a shovel that will immediately pick up 25 Dirt, I think it’s clear that statues can be made from the Excavator and Skid Steer. The same happens with Steam Tractor now.

**Comment by SlayksWood** [comment]
I thought and made a video for an example, how much it will take time.
![image](https://user-images.githubusercontent.com/45708377/64017585-9d566900-cb32-11e9-8ace-bbce11535e7b.png)
Steel Hoe:
https://drive.google.com/file/d/1wfEgGymPeTbMQVnVgf3rmhjTwvUzVCg_/view?usp=sharing
Steam tractor:
https://drive.google.com/file/d/1yHqrZJkd93KZWzLJ9LyPhEezp3oZ8EbV/view?usp=sharing

**Comment by WeaselDog** [comment]
At the very least the Steam Tractor should also till 5 horizontal spaces at a time, but I agree the hoe change is too much.

**Comment by elfl0rd** [comment]
Steam tractor is not spend calories...
But yes, i agree - some tractor things make it annoying.
https://github.com/StrangeLoopGames/EcoIssues/issues/12398
https://github.com/StrangeLoopGames/EcoIssues/issues/10009
https://github.com/StrangeLoopGames/EcoIssues/issues/12824

Also more things.
https://github.com/StrangeLoopGames/EcoSuggestions/issues/430

**Comment by SlayksWood** [comment]
After Last fix https://github.com/StrangeLoopGames/Eco/pull/5152
Need Balance for durability and Calories consumption

**Comment by johnkslg** [comment]
Agreed, new tools shouldn't make vehicles obsolete. I'd agree with @SlayksWood proposal.  In addition:
- Can make high level tools take more calories, since they till more area
- Can make high level tools require specialties to use 
- Can increase area that steam tractor affects.

**Comment by beastyn** [comment]
@SlayksWood has also a suggestion about tool's durability balance.
We can call it exactly Durability (not Uses), so we can remove N points for N blocks handling (even with one use)

For example, if you scythe 5 plant blocks in one use you will lose 5 durability points.

**Comment by SlayksWood** [comment]
Some suggestion for Scythe.
![image](https://user-images.githubusercontent.com/45708377/72596194-8b783580-391c-11ea-9f59-db939389ba62.png)

**Comment by Sekor2600** [comment]
This would be more realistic. I tink together with the changes that beastyn did post would this be better.
Maybe the Calories should then all count with each block that is effected too

**Comment by keegan-o** [comment]
Adding Simon to this as he is likely the one that would be able to work on the feature extension of changing the targeted/affected blocks into custom shape patterns.

**Comment by Sekor2600** [comment]
From Balance Channel:

- [ ] Make it that Tractor Plough alsways deletes Grass. Bigger Radius of affected Blocks

- [ ] Tools always use Calories and durability of sffectes Blocks (Example: Hoe affects 3 Block you use 3x Calories and 3x durability

- [ ] Use System from Slayks for the Scythe.

- [ ] 
![image](https://user-images.githubusercontent.com/37456949/73727473-cf977280-4731-11ea-9661-55ab6bd8959c.png) wooden 1 block. Iron, Steel 2 Blocks. Modern 3 Blocks for Hoe

**Comment by johnkslg** [comment]
Adding Quadbro, I think he will take this system from Simon

**Comment by redwyre** [comment]
Tool areas: https://docs.google.com/spreadsheets/d/1ysaXL08trZroPb2zpvIQ0-3cRFYGpw4miMjEileTL74/edit#gid=0

**Comment by SlayksWood** [comment]
@redwyre 
Maybe we should leave Steel sickle as a tool for one block? Seems that, sickle should not mow over the area. It has small size for this.
@keegan-o 
We either need a model for steel sickle (now the same with Scythe) or temporarily hide it.

**Comment by redwyre** [comment]
These are now specified in the excel file under "AreaBlocks", empty means just the selected block. I'm fine with it if someone wants to remove it for sickle. Just try to keep the spreadsheet above up to date to make it easier to visualize.

I'm going to remove myself from this now, but feel free to ping me if anyone needs anything.

**Comment by SlayksWood** [comment]
Seems almost all fixed, I created separate issue for steel sickle #14836
